sphinx/themes/basic/static/doctools.js

```diff
@@ -252,6 +253,30 @@ var Documentation = {
     });
     var url = parts.join('/');
     return path.substring(url.lastIndexOf('/') + 1, path.length - 1);
+  },
+
+  /**
+   * init onKeyListernes (for navigation)
+   */
+  initOnKeyListeners: function() {
+    $(document).keyup(function(event) {
+      if (!$(document.activeElement).is('input')) { //don't navigate when in search box
+        switch (event.keyCode) {
+          case 37: //left
```

Comment:
Small style nit:  The rest of the file uses `// comment` (with a space inbetween, see also above/below.)